submit-api/src/submit_api/enums/activity_type.py

```diff
@@ -0,0 +1,42 @@
+"""Submission item status class.
+
+Manages the item status
+"""
+from __future__ import annotations
+
+import enum
+
+
+class ActivityTypeEnum(enum.Enum):
+    """Enum for activity type statuses."""
+
+    SUBMISSION = 'SUBMISSION'
+    USER = 'USER'
+
+
+class ActorTypeEnum(enum.Enum):
```

Comment:
Does this mean staff or entity user? If so then the Staff here is also a user, maybe we can change to something like:
ENTITY_USER
STAFF_USER